helper_test.go

```diff
@@ -1161,6 +1164,45 @@ func createVariableSetVariable(t *testing.T, client *Client, vs *VariableSet, op
 	}
 }
 
+func waitForSVOutputs(t *testing.T, client *Client, svID string) {
+	t.Helper()
+	retry(func() (interface{}, error) {
```

Comment:
I think you should interpret the return error value of retry and possibly set t.Error or t.Fatal if it's not nil?